dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/NetUtilsTest.java

```diff
@@ -30,12 +30,13 @@
 
 import org.junit.After;
 import org.junit.Test;
-import org.powermock.reflect.Whitebox;
+import org.springframework.test.util.ReflectionTestUtils;
```

Comment:
@kezhenxu94 @ruanwenjun @caishunfeng Currently `springframework.test.util.ReflectionTestUtils` is the only thing I find that could replace `powermock.reflect.Whitebox`. However, it could not modify `final` field. That's why I remove `final` keyword of `KUBERNETES_MODE` in `Constant.java`. I'm not sure whether there is a better way.

Reply:
You don't need to set that field. You can use https://github.com/stefanbirkner/system-rules to mock the system env used here https://github.com/apache/dolphinscheduler/blob/66958b91d76d1b89502ff2e041a5eb2cadfbe5ac/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java#L800.

Reply:
> You don't need to set that field. You can use https://github.com/stefanbirkner/system-rules to mock the system env used here
> 
> https://github.com/apache/dolphinscheduler/blob/66958b91d76d1b89502ff2e041a5eb2cadfbe5ac/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java#L800
> 
> .

Thanks for the help. I will take a look into it  : )